nCompiler/R/compile_eigenization.R

```diff
@@ -143,10 +143,10 @@ compile_eigenize <- function(code,
             return(setupExprs)
         }
         if(code$name == '[') { ## If there is still A[i] in the code, it is because it is equivalent to a scalar and does not need eigenization
-            if(code$nDim == 0) {
+            if(code$type$nDim == 0) {
                 return(NULL)
             } else {
-                writeLines(paste0('Warning, in eigenizing ',nimDeparse(code), ' the [ is still there but nDim is not 0 (not a scalar).') )
+                writeLines(paste0('Warning, in eigenizing ',deparse(code$Rexpr), ' the [ is still there but nDim is not 0 (not a scalar).') )
```

Comment:
I think this should be `nDeparse(code)`

Reply:
Updated: https://github.com/nimble-dev/nCompiler/pull/3/commits/749aaf11b87d1a69a2947ce2c021a997451d061d